lib/riemann/proto.ex

```diff
@@ -1,12 +1,8 @@
 defmodule Riemann.Proto do
-  # this is broken, "from" isn't executed in the context of this module. :(
-  # @proto_file Path.expand("riemann.proto", __DIR__)
-  # @external_resource @proto_file
-  # use Protobuf, from: @proto_file
-
-  alias Riemann.Helpers
   use Protobuf, from: Path.expand("riemann.proto", __DIR__)
-  use_in "Event", Helpers.Event
-  use_in "Msg",   Helpers.Msg
-  use_in "Attribute", Helpers.Attribute
+  @external_resource Path.expand("riemann.proto", __DIR__)
```

Comment:
curious, i wonder if it's because the `from: ` value is really passed as AST, and maybe the module attribute isn't executed in the correct context